apps/shipments/iot_client.py

```diff
@@ -36,3 +39,13 @@ def update_shadow(self, device_id, shadow):
             raise AWSIoTError("Error in response from AWS IoT")
 
         return iot_shadow['data']
+
+
+class URLShortener(URLShortenerClient):
+    def get_shortened_link(self, params):
+        url_response = self._post(payload=params)
+
+        if 'short_id' not in url_response:
+            raise UrlShortenerError("Error generating short_url for Permission Link")
```

Comment:
Why do you have short_url underscored?

Reply:
That was originally when I was using the short_url from the payload in the return